beeline/src/java/org/apache/hive/beeline/BeeLine.java

```diff
@@ -835,7 +835,15 @@ private boolean connectUsingArgs(BeelineParser beelineParser, CommandLine cl) {
 
     driver = cl.getOptionValue("d");
     auth = cl.getOptionValue("a");
-    user = cl.getOptionValue("n");
+    /*
+    Fetching user from system properties to check who is loggedin
+     */
+//    user = cl.getOptionValue("n");
+    try {
+      user = ExtractShellUtils.getLoggedInUserFromShell();
```

Comment:
why catch throwable if it is a checked exception from the method? Catch exception and put it as runtime

Reply:
done